src/consensus/consensus.h

```diff
@@ -26,12 +26,9 @@ static const int64_t MAX_BLOCK_SIGOPS_COST = 80000;
 static const int COINBASE_MATURITY = 100;
 
 /** Flags for nSequence and nLockTime locks */
-enum {
-    /* Interpret sequence numbers as relative lock-time constraints. */
-    LOCKTIME_VERIFY_SEQUENCE = (1 << 0),
-
-    /* Use GetMedianTimePast() instead of nTime for end point timestamp. */
-    LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
-};
```

Comment:
Isn't an enum appropriate when it's indeed an enumeration?

Aso, please rebase, as there are other consts that could be constexprs in consensus.h now:

```c++
static const size_t MIN_TRANSACTION_WEIGHT = WITNESS_SCALE_FACTOR * 60; // 60 is the lower bound for the size of a valid serialized CTransaction
static const size_t MIN_SERIALIZABLE_TRANSACTION_WEIGHT = WITNESS_SCALE_FACTOR * 10; // 10 is the lower bound for the size of a serialized CTransaction
```

Still a bit confused about the whole thing, seems like simple `const` would also suffice. Is the advantage of `constexpr` just that it gives a compile time error in case the expression can't be evaluated at compile time?